templates/genericchart.html

```diff
@@ -63,6 +63,22 @@
         })
         {{else}}
         Highcharts.stockChart('chart', {
+            responsive: {
+            rules: [{
+                condition: {
+                maxWidth: 400,
```

Comment:
please indent maxWith

Reply:
@MauserBitfly I took a look, and we won't be able to make it more dynamic that what currently is. For a better handling of these buttons we would need to use a newer version of the highcharts js library. Imo since it is at least usable now, let's leave it like that, and we can consider an update of the library later.